app/controllers/posts_controller.rb

```diff
@@ -478,6 +478,7 @@ def permitted_params(include_associations=true)
       :authors_locked,
       :audit_comment,
       :private_note,
+      :editor_mode,
```

Comment:
Do we use the root params[:editor_mode] anywhere, or just params[:post][:editor_mode]? That's probably related to why we have the two hidden fields below, so maybe grepping for editor_mode would be useful